frontend/benefit/applicant/public/locales/fi/common.json

```diff
@@ -468,14 +468,22 @@
   },
   "select": "Valitse",
   "login": {
-    "login": "Kirjaudu palveluun",
+    "login": "Kirjaudu sisään",
     "logoutMessageLabel": "Olet kirjautunut ulos",
     "errorLabel": "Tapahtui tuntematon virhe. Kirjaudu uudelleen sisään",
     "sessionExpiredLabel": "Käyttäjäsessio vanhentui. Kirjaudu uudelleen sisään",
-    "infoLabel": "Palvelun käyttäminen edellyttää vahvaa tunnistautumista",
+    "infoLabel": "Ongelmia kirjautumisessa?",
     "logoutInfoContent": "Sinut on kirjattu ulos. Jos haluat jatkaa asiointia palvelussa, kirjaudu uudelleen sisään klikkaamalla alla olevasta painikkeesta.",
-    "infoContent": "Jotta voit edetä palvelussa, sinun täytyy tunnistautua. Voit valita itse, mitä tunnistustapaa haluat käyttää. Tunnistustietosi kulkevat suojatussa yhteydessä. Klikkaa alla olevasta painikkeesta tunnistautuaksesi.",
-    "termsOfServiceHeader": "Tietoa työnantajan edustajien henkilötietojen käsittelystä"
+    "infoContent": "Lähetä meille sähköposti osoitteeseen helsinkilisa@hel.fi.",
+    "termsOfServiceHeader": "Tietoa työnantajan edustajien henkilötietojen käsittelystä",
+    "header": "Kirjaudu Helsinki-lisän asiointipalveluun",
```

Comment:
`header` → `heading` and `smallHeader` → `subHeading` for consistency throughout the apps.